ops/scripts/generate_config.sh

```diff
@@ -122,9 +109,19 @@ TARGET=${APP_SOURCE}/protected/config/console.php
 VARS='$NONE'
 envsubst $VARS < $SOURCE > $TARGET
 
+
+if [[ $GIGADB_ENV != "dev" && $GIGADB_ENV != "CI" ]];then
+  export YII_TRACE_LEVEL=${YII_TRACE_LEVEL:-"0"}
+  export YII_DEBUG=${YII_DEBUG:-"false"}
+  export DISABLE_CACHE=${DISABLE_CACHE:-"false"}
+else
+  export YII_TRACE_LEVEL=${YII_TRACE_LEVEL:-"3"}
+  export YII_DEBUG=${YII_DEBUG:-"true"}
+  export DISABLE_CACHE=${DISABLE_CACHE:-"true"}
```

Comment:
`YII_DEBUG` is in `env-sample`. Do we need to add `YII_TRACE_LEVEL` and `DISABLE_CACHE` into `env-sample` too?

Reply:
@pli888, from a functional perspective it isn't required as the defaults are already set in the configuration script.
However from a documentation perspective it's better that they are present in ``env-sample``, so I've committed your suggested change.